public/404.html

```
@@ -1,67 +1,71 @@
-  <title>The page you were looking for doesn't exist (404)</title>
-  <meta name="viewport" content="width=device-width,initial-scale=1">
-  <style>
-  .rails-default-error-page {
-    background-color: #EFEFEF;
-    color: #2E2F30;
-    text-align: center;
-    font-family: arial, sans-serif;
-    margin: 0;
-  }
+  <head>
+    <title><404 - ページが見つかりません | SokoNote ></title>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </404> ], start tag match failed [ <404 - ページが見つかりません | SokoNote > ] on line 4.

(tag-pair)